connector/pom.xml

```diff
@@ -55,6 +55,13 @@
             <version>${flink.version}</version>
             <scope>provided</scope>
         </dependency>
+        <dependency>
+            <groupId>org.apache.flink</groupId>
+            <artifactId>flink-table-planner_${scala.binary.version}</artifactId>
+            <version>${flink.version}</version>
+            <type>test-jar</type>
+            <scope>test</scope>
+        </dependency>
```

Comment:
Repeated dependency？

Reply:
I added this `test-jar` as a test dependency so that we can use the `'values'` table connector when writing Flink integration tests. This connector allows us to build a table from test data in the code. I've seen this used in the official JDBC connector as well.

Reply:
I see it, got a new approach for testing.